### ISSUE TITLE
Newsletter

### ISSUE THREAD

**Issue by milancurcic** [fortran-lang/fortran-lang.github.io#3]
Let's discuss what should the fortran-lang newsletters be and how we should work on them.

I posted an example newsletter in #2 yesterday. You can see the post [here](https://fortran-lang.org/newsletter/2020/04/06/Announcing-FortranCon-2020/) and the source file [here](https://raw.githubusercontent.com/fortran-lang/fortran-lang.github.io/master/_posts/2020-04-06-Announcing-FortranCon-2020.md). With Jekyll, like with most popular static site generators, you write posts in plain markdown, with html for any extras.

## Topics

First, what topics/news should be covered by the newsletter. Here's my idea:

* Events: conferences, meetups, and similar. These could be either announcements for upcoming events or briefings/reports from events that happened. They should be related to Fortran. A NumPy or weather prediction conference is probably off-topic. Fortran or HPC meetings are on topic.
* Updates from J3 and WG5 meetings and developments of the language. Highlight proposals discussed. Like what @certik has been posting directly in issues in j3-fortran/fortran_proposals.
* Fortran compiler releases
* New development in fortran-lang projects: For example, highlighting new modules or functions in stdlib, or new fpm capability.
* New packages available in the fpm registry.
* Last but not least, new Fortran releases (202x, 202y...). This one's pretty obvious. :)
* Anything else?

What I think should be off-topic and omitted in newsletters:

* Personal (non-community) Fortran projects, except listing them when they're added to the fpm registry.
* News not directly about Fortran but that could be interesting to Fortran developers. Stuff about other languages etc.

## Format

I think we could have two kinds of newsletters:

* Off-schedule, topic-specific. This kind of newsletter covers only one item that is noteworthy. Examples: Fortran meetings, language releases, perhaps even compiler releases.
* Monthly, recurring. This kind of newsletter comes out periodically at the beginning of each month, summarizing the Fortran development and news from the previous month. It's formatted more as a summary rather than an in-depth coverage of a topic, so it can include on-topic odds and ends.

## How should we work on it

Here's what I propose:

* Anybody can open a PR for a newsletter
* Must assign a reviewer/editor. Can assign more than one if desired, ideally not too many. You don't want to have to wait for 5 people to review if you want timely news to go out.
* Of course, anybody can comment. If somebody thinks a PR doesn't make appropriate newsletter, they should say so.
* For a monthly recurring newsletter, I suggest we open a draft PR at the beginning of each month, and we work on it as the month goes and new stuff happens. At the end of the month, we clean it up and merge.

Obviously, my PR #2 was not an example of this workflow. I just wanted to get something up quickly to use as an example.

I will do some research how we can automate posting of new newsletters to the @fortranlang Twitter automatically, perhaps via GitHub Actions. For people who use RSS readers, we have a feed on the website.

What do you think? @certik @jvdp1 @ivan-pi @everythingfunctional @jacobwilliams @marshallward @zbeekman

**Comment by certik** [comment]
I agree with everything you wrote. This is a good start. As we go, we can further refine or relax some of the rules.

**Comment by everythingfunctional** [comment]
That sounds well thought out and reasonable. I like it.

**Comment by milancurcic** [comment]
I just thought it'd also be useful for the monthly newsletter to have a list of Fortran job postings.

**Comment by certik** [comment]
The first newsletter is on the way at #4. Before we close this issue, let's create a document with information how the newsletter works. @milancurcic do you want to take a stab at that?

**Comment by awvwgk** [comment]
I created a wiki page on the monthly newsletter [here](https://github.com/fortran-lang/fortran-lang.org/wiki/Monthly-Newsletter). Summarizing parts of the workflow, contributions are welcome.
